src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -83,39 +91,56 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
     String fieldName = label != null && !label.isEmpty() ? label : name;
 
     int sqlType = metadata.getColumnType(col);
+
+    String defaultValue = whitelistDefaultValue(columnDefaults.get(name));
```

Comment:
Does Map.get() return null or blow up with an exception if the value isn't found?

Reply:
Probably want getOrDefault (or some other sort of error handling)
https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#getOrDefault-java.lang.Object-V-

Reply:
Map.get() will return null if there is no mapping for the key

Reply:
Works for me.